modules/claims_api/app/services/claims_api/service.rb

```diff
@@ -5,5 +5,66 @@ class Service
     def self.process(**args)
       new(args).process
     end
+
+    def log_service_progress(claim_id, tag, detail)
+      log_tag = tag == 'pdf' ? '526_v2_PDF_Generator_job' : '526_v2_Docker_Container_job'
+      ClaimsApi::Logger.log(self.class,
+                            log_tag:,
+                            claim_id:,
+                            detail:)
+    end
+
+    protected
+
+    def save_auto_claim!(auto_claim, status)
+      auto_claim.status = status
+      auto_claim.validation_method = ClaimsApi::AutoEstablishedClaim::VALIDATION_METHOD
+      auto_claim.save!
+    end
+
+    def set_established_state_on_claim(auto_claim)
+      save_auto_claim!(auto_claim, ClaimsApi::AutoEstablishedClaim::ESTABLISHED)
+    end
```

Comment:
@stiehlrod  Looks like this method is not used in the sync work anymore is it?  Looks like these all might not be used anymore, unless I am overlooking something?

- set_established_state_on_claim
- clear_evss_response_for_claim
- set_pending_state_on_claim
- established_state_value
- pending_state_value
- errored_state_value